physionet-django/user/models.py

```diff
@@ -827,6 +827,7 @@ class AutoRejectionReason(models.TextChoices):
     responder_comments = models.CharField(max_length=500, default='',
                                           blank=True)
     revoked_datetime = models.DateTimeField(null=True)
+    reminder_datetime = models.DateTimeField(null=True)
```

Comment:
It isn't clear that this relates to the reference, rather the applicant.

Reply:
Ah yes, thats true, Let me work on the changes